lib/commands/calculate.js

```diff
@@ -56,6 +57,11 @@ async function calculate({
   let durationMeans = successfulRanges.map(range => range.SUCCESSFUL.buildDurationMean);
   let { min, max } = math.getMinMax(durationMeans);
 
+  // return if getJson is true, avoiding logging in this case to make api faster
```

Comment:
Actually, I think you should log this output. That allows it to be used when piping between commands.

It would be nice if we could support using build-stats as a library at the same time though. But I would return an object instead of a string.